packages/atomic/src/button/LocalizeButton.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { lighten } from 'polished';
+import styled from '@emotion/styled';
+import classnames from 'classnames';
+
+import {
+  LocalizeThemeProps,
+  LocalizeProps,
+  LocalizeSize,
+} from '@seolhun/localize-components-styled-types';
+
+const DEFAULT_CLASSNAME = '__Localize__Button';
+type ButtonProps = React.HTMLAttributes<HTMLButtonElement>;
+
+export interface LocalizeButtonProps extends LocalizeProps, ButtonProps {
+  size?: LocalizeSize;
+
+  borderRadius?: string;
+}
+
+const getSizeStyle = (size?: LocalizeSize) => {
+  switch (size) {
+    case 'xl':
+      return '1.4rem 2rem';
+    case 'lg':
+      return '1.2rem 1.8rem';
+    case 'md':
+      return '1rem 1.4rem';
+    case 'sm':
+      return '0.8rem 1rem';
+    default:
+      return '0.6rem 0.8rem';
+  }
+};
+
+const StyledLocalizeButton = styled.button<
+  LocalizeButtonProps,
+  LocalizeThemeProps
+>(({ theme, size = 'md', bgColor = 'primary', borderRadius }) => {
+  const fonts = theme.fonts.font1;
+  const backgroundColor = theme.colors[bgColor];
```

Comment:
Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/manual/shun10114/localize-components/pullRequest?prid=5930444)